packages/vats/decentral-psm-config.json

```diff
@@ -0,0 +1,56 @@
+{
+  "bootstrap": "bootstrap",
+  "defaultReapInterval": 1000,
+  "vats": {
+    "bootstrap": {
+      "sourceSpec": "@agoric/vats/src/core/boot-psm.js",
+      "parameters": {
+        "anchorAssets": [
+          {
+            "denom": "ibc/usdc1234"
```

Comment:
what is this value? might be worth a RAEDME.md in this dir to explain.

looking at how this gets loaded, must it be JSON or could we use `.js` to get comments? https://github.com/Agoric/agoric-sdk/blob/eb5b1f5b0635680081d708e1928c8ee089b61c80/packages/cosmic-swingset/src/chain-main.js#L361-L365

Reply:
I think it has to be JSON.

It's a **denom** in the sense of Cosmos-SDK [ADR 024: Coin Metadata](https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-024-coin-metadata.md) for the anchor/reference asset that a PSM trades for IST.

See also [makeAnchorAsset](https://github.com/Agoric/agoric-sdk/blob/ba1ed62062a63862e2eecb598b0bd1d2ac828e1f/packages/inter-protocol/src/proposals/startPSM.js#L159-L166) and [bankManager.addAsset](https://github.com/Agoric/agoric-sdk/blob/ba1ed62062a63862e2eecb598b0bd1d2ac828e1f/packages/vats/src/vat-bank.js#L320-L328)